CMakeLists.txt

```
@@ -82,6 +82,11 @@ if(BUILD_WITH_MARCH_NATIVE)
   add_compile_options(-march=native)
 endif()
 
+if(WIN32)
```

```suggestion
if(MSVC)
```
Can be made more specific to not break potential MinGW support on Windows.